CONTRIBUTORS.rst

```diff
@@ -0,0 +1,20 @@
+.. _contributors:
+
+Contributors
+------------
+
+We follow the |icon| `all-contributors`_ specification.
+Contributions of any kind to ``pyvista`` are welcome!
+
+It's important that we recognise and acknowledge **all** our contributors, not just the ones who push code.
+So no matter how large or small, we're grateful for your contribution and caring enough to make a difference.
+
+- Maksym Kalaidov (`@mkskalai <https://github.com/mkskalai>`_) (Promotion)
+
+.. note::
+    If you have contributed to ``pyvista`` and your name is not listed here,
+    please submit a pull request to add yourself to the list.
+    Be sure to follow the `all-contributors`_ specification.
```

Comment:
Knowing the original (intended) setup here, this might be a bit confusing to users who are more familiar with `all-contributors`, exactly because we're using full text instead of emoji. Do you think we should try to specify in this sentence that we only need the names listed on the link?